.editorconfig

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Meaning, do we need to fix up the style on other Python code to match what this pep8 config would enforce? yes ideally so. Much of the code in the docs are actually broken out as source files that are inlined, so hopefully they can be tested directly by pep8. For stuff that isn't otherwise tested, yes, making it match the enforced style is worthwhile.